novawallet/Modules/Notifications/Management/NotificationsManagementPresenter.swift

```diff
@@ -18,9 +18,11 @@ final class NotificationsManagementPresenter {
     private var modifiedTopicsSettings: PushNotification.TopicSettings?
 
     private var isSaveAvailable: Bool {
-        (settings != modifiedSettings) ||
+        let settingsWasModified = (settings != modifiedSettings) ||
             (notificationsEnabled != modifiedNotificationsEnabled) ||
             (topicsSettings != modifiedTopicsSettings)
+        let isAmbiguousState = areAllNotificationsOff() == true && modifiedNotificationsEnabled == true
```

Comment:
Why do we need to check ambiguousState if we always check whether to set modifiedNotificationsEnabled to false when anything changes?